public/locales/es/graph.json

```diff
@@ -7,5 +7,6 @@
     "saturday":"S",
     "sunday": "D",
     "noSensor": "¡Haz clic en un sensor para ver sus datos!",
-    "time": "Tiempo"
+    "time": "Tiempo",
+    "graphTitle": "(Air Quality Index, AQI) en las últimas 24 horas"
```

Comment:
I would recommend using the translation for AQI on this as well as the dial?  That is the Índice de...

Reply:
I think we could just use AQI here too since AQI is defined in the gauge component

Reply:
I’m cool with that too, but just am not a fan of the parenthetical only